css/glossary-audio/glossary-audio-app.less

```diff
@@ -0,0 +1,24 @@
+.glossaryAudioApp {
+  color: #3f3f3f;
+  font-family: lato, arial, helvetica, sans-serif;
+
+  .logo {
+    height: 35px;
+  }
+
+  h1 {
+    padding: 15px;
+    background-color: #b7e2ec;
+    display: flex;
+    align-items: center;
+    gap: 10px;
+  }
+
+  .contents {
+    padding: 15px;
+
+    .info {
+      margin-bottom: 10px;
+    }
+  }
+}
```

Comment:
It's mad about no last line

Reply:
Fixed in https://github.com/concord-consortium/portal-report/pull/473/commits/8b9484765c04fe47b8b24a049023649c4564cace